src/Compiler/TypedTree/TypedTree.fs

```diff
@@ -4556,28 +4556,37 @@ type TupInfo =
 type Measure = 
 
     /// A variable unit-of-measure
-    | Var of typar: Typar
+    | Var of typar: Typar * range: range
```

Comment:
Is the range ever different from typar.Range ?

Reply:
Good spot. Yeah no need to have a dedicated range here.